src/search.rs

```diff
@@ -212,18 +241,24 @@ impl SearchArgs {
       return Ok(());
     }
 
-    let hyperlinks = supports_hyperlinks::supports_hyperlinks();
-    debug!(?hyperlinks);
-
     let nixpkgs_path_output = nixpkgs_path.join().map_err(|e| {
       color_eyre::eyre::eyre!("nixpkgs_path thread panicked: {e:?}")
     })?;
+    debug!("{:?}", nixpkgs_path_output);
```

Comment:
```suggestion
    debug!("{nixpkgs_path_output:?}");
```

Clippy will warn about this. I eludes me quite often as well.